src/filesystem/browser/filesystem-commands.ts

```diff
@@ -91,7 +98,23 @@ export class FileCommandContribution implements CommandContribution {
                 registry.registerCommand(openWithCommand);
                 registry.registerHandler(openWithCommand.id,
                     new FileSystemCommandHandler(this.selectionService, uri => {
-                        return opener.open(uri);
+
+                    })
+                );
+            }
+        });
+
+        this.openerService.getOpeners().then(openers => {
```

Comment:
Please roll back this file, it is the common fs extension, not cpp specific. What are you trying to achieve? You should do it in the cpp extension, via a new opener of a resource. Look at the java extension.